Tests/FluentAssertions.Specs/Exceptions/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -874,16 +788,16 @@ public void When_async_method_throws_the_expected_inner_exception_it_should_succ
             };
 
             // Act
-            Action action = () => task
-                .Should().Throw<AggregateException>()
-                .WithInnerException<InvalidOperationException>();
+            Func<Task> action = () => task
+                .Should().ThrowAsync<AggregateException>()
+                .WithInnerException<AggregateException, InvalidOperationException>(); // TODO How to remove duplicate generic argument?
```

Comment:
I think that would require some sort of [partial type inference](https://github.com/dotnet/csharplang/issues/1349)

Reply:
I've no idea...

Reply:
"partial type inference" seams to be a feature in (far) future. How can this help today?
I'll try again to find another solution. Do you have any suggestion?

Reply:
My comment about partial type inference was to indicate that, until we might see that feature in the language, I can't think of a prettier way to implement that feature.
So if we can't make it better, there's no TODO and the note can be removed.

Reply:
removed